app/cdap/components/MarketPlaceEntity/index.js

```diff
@@ -173,7 +173,7 @@ export default class MarketPlaceEntity extends Component {
               <button
                 className="btn btn-primary"
                 onClick={() => this.setState({ performSingleAction: true })}
-                data-cy={`${this.state.entityDetail.actions[0].type}-btn`}
```

Comment:
nit: is this `cy` attribute removed intentionally? please double check if so.

Reply:
This may have been removed (incorrectly) while I was cleaning up the PR. Added it back.